upgrade.php

```diff
@@ -110,42 +154,42 @@
     $git_checkout = shell_exec('git checkout '.$branch);
     $git_stash = shell_exec('git stash');
     $git_pull = shell_exec('git pull');
-    echo '-- '.$git_fetch;
+    echo $git_fetch;
     echo '-- '.$git_stash;
     echo '-- '.$git_checkout;
-    echo '-- '.$git_pull;
+    echo '-- '.$git_pull."\n";
 } else {
     echo "Git is NOT installed. You can still use this upgrade script to run common \n";
     echo "migration commands, but you will have to manually download the updated files. \n\n";
-}
+    echo "Please note that this script will not download the latest Snipe-IT \n";
+    echo "files for you unless you have git installed. \n";
+    echo "It simply runs the standard composer, artisan, and migration \n";
+    echo "commands needed to finalize the upgrade after. \n\n";
 
+}
 
 
 echo "--------------------------------------------------------\n";
 echo "Step 5: Cleaning up old cached files:\n";
 echo "--------------------------------------------------------\n\n";
 
-
-if (file_exists('bootstrap/cache/compiled.php')) {
-    echo "-- Deleting bootstrap/cache/compiled.php. It is no longer used.\n";
-    @unlink('bootstrap/cache/compiled.php');
-} else {
-    echo "-- No bootstrap/cache/compiled.php, so nothing to delete.\n";
-}
-
-if (file_exists('bootstrap/cache/services.php')) {
-    echo "-- Deleting bootstrap/cache/services.php. It it no longer used.\n";
-    @unlink('bootstrap/cache/services.php');
-} else {
-    echo "-- No bootstrap/cache/services.php, so nothing to delete.\n";
-}
-
-if (file_exists('bootstrap/cache/config.php')) {
-    echo "-- Deleting bootstrap/cache/config.php. It it no longer used.\n";
-    @unlink('bootstrap/cache/config.php');
-} else {
-    echo "-- No bootstrap/cache/config.php, so nothing to delete.\n";
+// Build an array of the files we generally want to delete because they
+// can cause issues with funky caching
+$unused_files = [
+    "bootstrap/cache/compiled.php",
+    "bootsrap/cache/services.php",
+    "bootstrap/cache/config.php",
+];
+
+foreach ($unused_files as $unused_file) {
+    if (file_exists($unused_file)) {
+        echo "√ Deleting ".$unused_file.". It is no longer used.\n";
+        @unlink('bootstrap/cache/compiled.php');
```

Comment:
Whoops! I think you want to `@unlink($unused_file)` here

Reply:
Whoops, good catch!